module/VuFindSearch/src/VuFindSearch/Backend/Blender/Response/Json/RecordCollection.php

```diff
@@ -0,0 +1,474 @@
+<?php
+
+/**
+ * JSON-based record collection for records from multiple sources.
+ *
+ * PHP version 7
+ *
+ * Copyright (C) The National Library of Finland 2022.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Search
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     http://vufind.org
+ */
+namespace VuFindSearch\Backend\Blender\Response\Json;
+
+use VuFindSearch\Backend\EDS\Response\RecordCollection as EDSRecordCollection;
+use VuFindSearch\Backend\Solr\Response\Json\Facets;
+use VuFindSearch\Response\RecordCollectionInterface;
+
+/**
+ * JSON-based record collection for records from multiple sources.
+ *
+ * @category VuFind
+ * @package  Search
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     http://vufind.org
+ */
+class RecordCollection
```

Comment:
Should be no need, this is fully covered by the tests in BackendTest.

Reply:
In theory it would probably be nice to have a separate test for this class and to use more mocking in the BackendTest... but as we discussed elsewhere, getting coverage is the first priority, and we can worry about being perfectionists about test organization later. :-)